.travis.yml

```diff
@@ -4,8 +4,8 @@ dist: trusty
 
 env:
     global:
-    - BYOND_MAJOR="511"
-    - BYOND_MINOR="1385"
+    - BYOND_MAJOR="512"
+    - BYOND_MINOR="1424"

```

Comment:
Drop these changes, as we'll be going higher than 1424, and do it via a different PR.